models/spike_detector.h

```diff
@@ -183,6 +183,7 @@ class spike_detector : public Node
   Buffers_ B_;
 
   bool user_set_precise_times_;
+  long user_set_precision_value;
```

Comment:
`user_set_precision_value` is a private member. Therefore, the name should end in a `_`.